### PR TITLE
Make overlap-profile-photo work when the photo is not the direct child

### PR DESCRIPTION
**Pull request by chinaowl**

### CODE REVIEW

File: dist/forum-design-system.css [hunk 1]
```diff
@@ -1109,6 +1109,15 @@ a.underlined-link, .underlined-links a {
...
 
+.profile-photo-overlap *:not(:first-child) .profile-photo-large {
+  margin-left: -12px; }
+
+.profile-photo-overlap *:not(:first-child) .profile-photo-medium {
+  margin-left: -8px; }
+
+.profile-photo-overlap *:not(:first-child) .profile-photo-small {
+  margin-left: -4px; }
+
```
Comment (awdyson) [inline dist/forum-design-system.css:1120]: If we're gonna allow this, maybe we should remove the direct-child (`>`) versions?

File: src/_profile-photos.scss [hunk 2]
```diff
@@ -45,10 +45,26 @@
...
+    }
+
+    .profile-photo-medium {
+      margin-left: -12px;
+    }
+
+    .profile-photo-small {
+      margin-left: -12px;
+    }
+  }
```
Comment (awdyson) [inline src/_profile-photos.scss:69]: The output looks correct, but I'm not sure why 😅
Shouldn't these be -12, -8, and -4, respectively?

File: dist/forum-design-system.css (hunk 1 above)
Comment (chinaowl) [reply dist/forum-design-system.css:1120]: Hmm, I thought it would be okay to allow it to work with both?

File: src/_profile-photos.scss (hunk 2 above)
Comment (chinaowl) [reply src/_profile-photos.scss:69]: Oops, good catch

File: dist/forum-design-system.css (hunk 1 above)
Comment (awdyson) [reply dist/forum-design-system.css:1120]: What I mean is that your change is a strictly better version of the rule for our given needs.
No reason to keep the old rule around.
Not a huge deal though, if you don't want to run another release.

**Pull request merged**